common/arch/win32/rbaudio.cpp

```diff
@@ -392,92 +417,69 @@ int RBAPeekPlayStatus()
     mciStatusParms.dwItem = MCI_STATUS_MODE;
     if ((mciError = mciSendCommand(wCDDeviceID, MCI_STATUS, MCI_STATUS_ITEM, reinterpret_cast<DWORD_PTR>(&mciStatusParms))))
     {
-		Warning("RBAudio win32: cannot determine MCI media status (%lx)", mciError);
+        Warning("RBAudio win32/MCI: cannot determine MCI media status (%lx)", mciError);
         return 0;
     }
 
-	if (mciStatusParms.dwReturn == MCI_MODE_PLAY)
+    if (mciStatusParms.dwReturn == MCI_MODE_PLAY)
         return 1;
-	else if (mciStatusParms.dwReturn == MCI_MODE_PAUSE)
-		return -1; // hack so it doesn't keep restarting paused music
-	else
-		return 0;
+    else if (mciStatusParms.dwReturn == MCI_MODE_PAUSE)
+        return -1; // hack so it doesn't keep restarting paused music
+    else
+        return 0;
 }
 
 static int cddb_sum(int n)
 {
-	int ret;
+    int ret;
```

Comment:
Please don't change indentation unless (1) the old layout is visibly confusing or (2) you are changing the surrounding logic such that a new indentation makes more sense (such as adding or removing a level of control flow statements, and reindenting the guarded block(s) accordingly).

Indentation should be done using tabs, with tabstops at 4 spaces.